client/src/services/cryptpad.ts

```diff
@@ -158,23 +159,51 @@ export class Cryptpad {
 }
 
 export function getDocumentTypeFromExtension(extension: string): CryptpadDocumentType {
+  // CryptpadDocumentType.Pad adds html content to text files, avoiding using it for now
+  // CryptpadDocumentType.Presentation works with .pptx, but is disabled as long as there's no viewer for it
   switch (extension.toLowerCase()) {
+    case 'xml':
```

Comment:
Why not use what's in `fileTypes.ts`? We have two different files for the same thing. We have the whole `detectFileType` that's used for the viewers, why not use this to choose what editor to use?